packages/Tag/src/Tag.js

```diff
@@ -1,28 +1,40 @@
 import React from "react";
```

Comment:
Your changes to the Tag component though are more significant and should be added to the `CHANGELOG`.